http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -4,14 +4,23 @@ set -e
 
 # If you update this file, please update azure.custom-script-extension.file-uris in reference.conf so that Leonardo can adopt the new script
 
+# In order to avoid user id conflicts, change azureuser uid and gid before jupyter or welder users are created
```

Comment:
Technically, the welder-user is never created here, and only exist in the welder container, which is why this hack is working :)

Reply:
Could you add a line before the welder docker run to explain that the DSVM WORKING_DIR owned by jupyter-user 1001 gets mounted as the Welder container work directory owned by user 1001 (which luckily is welder-user in there, but this logic is brittle and we will fix it later)